week-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,75 @@
-function setAlarm() {}
+const timeRemaining = document.querySelector('#timeRemaining');
+const circularProgressBar = document.querySelector("#circularProgressBar");
+let time;
+let input;
+let timeOut;
+let multiplierFactor;
+function setAlarm() {
```

Comment:
What happens if I set the first alarm. And while the first alarm is still counting / running, I set another alarm. How would your code behave in that situation?